### PR TITLE
Pin rrplug version to `3.0.0`

### PR DESCRIPTION
**Pull request by catornot**
also fixes things because of rrplug changes :P

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-rrplug = { git = "https://github.com/R2NorthstarTools/rrplug" }
+rrplug = "=3.0.0"
```
Comment (GeckoEidechse) [inline Cargo.toml:9]: Is there a reason to do `"=3.0.0"` vs just `"3.0.0"`? ^^"

**Review by GeckoEidechse — COMMENTED** [review]
Probably should get someone to test this and then should be good to merge. I see two smaller unrelated changes that I'm gonna quickly do myself.

As a sidenote, we should do like format check and clippy on CI for DiscordRPC ^^

File: Cargo.toml (hunk 1 above)
Comment (catornot) [reply Cargo.toml:9]: I don't quite follow semver so I might break smth accidentally in the future :>

Comment (GeckoEidechse) [reply Cargo.toml:9]: I thought `3.0.0` already means `=3.0.0`? ^^"

Comment (catornot) [reply Cargo.toml:9]: iirc it means `3.0.*`

**Review by GeckoEidechse — APPROVED** [review]
Confirmed working in testing. Joined a few servers and saw status update in Discord.

File: Cargo.toml (hunk 1 above)
Comment (GeckoEidechse) [reply Cargo.toml:9]: Huh, TIL. Apparently it's even 

```
3.0.0  :=  >=3.0.0, <4.0.0
```

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html


So yeah, in that case pinning is totally fine as rrplug version follows Northstar plugins version with major version number meaning minor changes could break compat within rrplug.

**Pull request merged**